src/js/utils/html-utils.js

```diff
@@ -0,0 +1,114 @@
+/**
+ * Useful functions for dealing with HTML.
+ *
+ * In particular, these functions default to being safe against
+ * Cross Site Scripting (XSS) attacks.
+ */
+(function(define) {
+    'use strict';
+    define(['underscore', './string-utils.js'], function(_, StringUtils) {
+        var escape, interpolateHtml, HTML;
+
+        /**
+         * Creates an HTML snippet.
+         *
+         * The intention of an HTML snippet is to capture a string that
+         * is known to contain HTML that has been safely escaped.
+         * As an example, this allows interpolate to understand that
+         * it does not need to further escape this HTML.
+         *
+         * @param {string} htmlString The string of HTML.
+         * @constructor
+         */
+        function HtmlSnippet(htmlString) {
+            this.text = htmlString;
+        }
+        HtmlSnippet.prototype.toString = function() {
+            return this.text;
+        };
```

Comment:
It could also be useful to implement `valueOf`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/valueOf